dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -389,8 +389,8 @@ public Workflow refresh(@ApiParam(hidden = true) @Auth User user,
         return workflow;
     }
 
-    @GET
-    @Path("/upsertVersion/")
+    @PUT
+    @Path("/path/workflow/upsertVersion/")
```

Comment:
If you're going to use `/path/workflow`, the pattern should be `/path/workflow/{repository}/upsertVersion`, and then remove organization name and repository query parameters. Otherwise, I think you should just go back to  `/upsertVersion`. And I'd probably change it from `upsertVersion` to `versions`, e.g., `/path/workflow/{repository}/versions`, although I'm not sure on that one.

If you're trying to get this in to the release, I'm OK doing it another PR.

Reply:
Oops I misunderstood your last comment, this makes a lot more sense. I think I'll just make the changes quickly and get them into this release/deployment.